microprofile/grpc/core/src/main/java/io/helidon/microprofile/grpc/core/Instance.java

```diff
@@ -129,15 +132,28 @@ public T get() {
             return ensureInstance();
         }
 
-        private synchronized T ensureInstance() {
-            if (instance == null) {
-                try {
-                    instance = instanceClass.newInstance();
-                } catch (Throwable e) {
-                    throw Status.INTERNAL.withCause(e).asRuntimeException();
+        private T ensureInstance() {
+            return accessInstance(() -> {
+                if (instance == null) {
+                    try {
+                        instance = instanceClass.newInstance();
```

Comment:
`newInstance()` is deprecated and doesn't throw `Throwable`.  I recognize this is not your code.

Reply:
Noted